### PR TITLE
Close #3489: bullet list rendered "opened up"

### LINKED ISSUE #3489: bullet list rendered "opened up"
Subject: why is some list at top of CHANGES rendered with paragraphs ?

### Problem

http://www.sphinx-doc.org/en/master/changes.html#deprecated
renders "opened up". I don't understand why from reST source, it does not look
different from similar mark-up elsewhere.

#### Error logs / results

```
<h3>Deprecated<a class="headerlink" href="#deprecated" title="Permalink to this headline">¶</a></h3>
<ul>
<li><p class="first"><code class="docutils literal"><span class="pre">sphinx.util.compat.Directive</span></code> class is now deprecated. Please use instead
<code class="docutils literal"><span class="pre">docutils.parsers.rsr.Directive</span></code></p>
</li>
```
#### Expected results
list items do not use ``<p>`` tags

### Environment info
- Sphinx version: master branch

Apologies if I am just ignorant of some reStructuredText spec.

### PR DESCRIPTION
**Pull request by jfbu**
Subject: fix description of ``html_compact_lists`` config setting.

### Feature or Bugfix
- Bugfix

Relates #3489

### CODE REVIEW

**Review by tk0miya — APPROVED** [review]

File: doc/config.rst
Comment (jfbu) [inline doc/config.rst]: I think I need to say "a list", not "lists"

File: doc/config.rst
Comment (jfbu) [inline doc/config.rst]: (cont.) else here ending should be "for any of their items".

**Comment by jfbu** [comment]
Thanks for review @tk0miya ! I fixed grammar (I hope, I am not native English speaker) and will merge once approved by Travis.

DocUtils concept of simple list is completely unstable because a single addition to one item in a deeply nested list propagates back up to the top list and all its items.

Should we worry about the mark-up in CHANGES (http://www.sphinx-doc.org/en/master/changes.html#deprecated) to keep all its lists "simple" in DocUtils sense ?

**Comment by tk0miya** [comment]
No, we have already many non-simple lists in CHANGES.

**Pull request merged**
